FXyz-Importers/src/main/java/org/fxyz3d/importers/obj/ObjImporter.java

```diff
@@ -399,24 +399,24 @@ protected void parseFace(String line) {
             }
         }
 
-        private void parseSmoothGroup(String line) {
-            currentSmoothGroup = line.substring(2).equals("off") ? 0 : Integer.parseInt(line.substring(2));
+        private void parseSmoothGroup(String value) {
+            currentSmoothGroup = value.equals("off") ? 0 : Integer.parseInt(value.substring(2));
```

Comment:
`value.substring(2)` is wrong, should be just `value`.

Reply:
Good catch, I missed that one.